src/.vuepress/plugins/update-handlebars-cli-help.js

```diff
@@ -29,17 +29,17 @@ export function updateHandlebarsCliHelp() {
         token.map = [startLine, startLine + 1];
         return true;
       });
-    }
+    },
   };
 }
 
 function getHandlebarsHelpFromCli() {
   const handlebarsCli = require.resolve("handlebars/bin/handlebars");
   const nodeExecutable = process.argv[0];
-  const { stderr } = cp.spawnSync(nodeExecutable, [handlebarsCli, "--help"], {
+  const { stdout } = cp.spawnSync(nodeExecutable, [handlebarsCli, "--help"], {
     argv0: "handlebars",
     stdio: ["pipe", "pipe", "pipe"],
-    encoding: "utf-8"
+    encoding: "utf-8",
   });
-  return stderr.replace(/^Usage: \S+ \S+/m, "handlebars");
```

Comment:
I'm not sure why this string-replace was done... it remove important info from the help, so I removed it.